pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -52,14 +52,16 @@ func (r *resourceAllocationScorer) score(
 	if r.resourceToWeightMap == nil {
 		return 0, framework.NewStatus(framework.Error, "resources not found")
 	}
-	requested := make(resourceToValueMap, len(r.resourceToWeightMap))
-	allocatable := make(resourceToValueMap, len(r.resourceToWeightMap))
+	requested, allocatable := make(resourceToValueMap), make(resourceToValueMap)
```

Comment:
why not a line for each? Better code-style IMO

Reply:
I don't have a strong option. Reverted.